src/components/grid/grid.ts

```diff
@@ -334,6 +335,10 @@ export class Grid extends Component<Props, SpreadsheetChildEnv> {
     return this.hoveredCell.col === col && this.hoveredCell.row === row;
   }
 
+  get gridRect(): Rect {
```

Comment:
I'm not a great fan of this. We have to "trimballer" `gridRect` from the top component to every children and children of children to be sure that **IF** a child use popover, this popover will correctly placed. It makes me feel that we introduce here a coupling between all our components.
It's typically a good use case for `useChildSubEnv`, you can add a function `popoverContainerRect` in children env, it will be automatically propagated and use by the popover component directly. With that, intermediate components should not take care of "trimballer" the props.